litmus/skel.ml

```diff
@@ -1090,8 +1094,6 @@ module Make
                            (CType.dump (register_type rloc (CType.Base t)))
                            sloc k::pp_rec (k+1) in
                      String.concat "," (pp_rec 0)
-                 | t when CType.is_ins_t t ->
-                    sprintf "pretty_opcode(o[%s_f])" sloc
```

Comment:
Removing those lines looks problematic. For instance, **litmus7** now produces C code that does not compile on test [L049](https://github.com/herd/herdtools7/blob/master/herd/tests/instructions/AArch64/L049.litmus).
Restoring the deleted lines apparently fixes the anomaly.

Reply:
Here is a suggestion of commit: ff8738d72443536774c96ba994463512475ca92c, as the "suggest" button is not available for a deletion :)